test/spec/modules/sharethroughBidAdapter_spec.js

```diff
@@ -15,7 +16,12 @@ const bidRequests = [
     userId: {
       tdid: 'fake-tdid',
       pubcid: 'fake-pubcid',
-      idl_env: 'fake-identity-link'
+      idl_env: 'fake-identity-link',
+      id5id: 'fake-id5id',
```

Comment:
as noted above, this is not what the `id5id` looks like, so you'll want to update it to ensure your tests are working properly.